pom.xml

```diff
@@ -83,6 +102,44 @@
       </modules>
     </profile>
 
+    <profile>
+      <id>codecov</id>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
I think codecov might always fail for this project; maybe we can consider omitting it for this project?

This project doesn't have any unit tests since we are just registering classes as using reflection for each library via the GraalVM SDK; i.e.: https://github.com/GoogleCloudPlatform/google-cloud-graalvm-support/blob/master/google-cloud-graalvm-support/src/main/java/com/google/cloud/graalvm/features/core/GrpcNettyFeature.java#L57

It's tested in the CI by building the samples as native images and then running them, seeing if we get any errors during compilation or runtime.